extensions/adhan-time/src/adhan.tsx

```diff
@@ -0,0 +1,47 @@
+import { MenuBarExtra, getPreferenceValues, Icon, openCommandPreferences, Cache } from "@raycast/api";
+import { useFetch } from "@raycast/utils";
+export interface Preferences {
+  country: string;
+  city: string;
+  calculation_methods: string;
+}
+export interface Data {
+  data: {
+    timings: {
+      Fajr: string;
+      Sunrise: string;
+      Dhuhr: string;
+      Asr: string;
+      Maghrib: string;
+      Isha: string;
+    };
+  };
+  calculation_methods: string;
+}
+export default function Command() {
+  const userPreference: Preferences = getPreferenceValues();
+  const { isLoading, data } = useFetch<Data>(
+    `https://api.aladhan.com/v1/timingsByCity?city=${userPreference.city}&country=${userPreference.country}&method=${userPreference.calculation_methods}`,
```

Comment:
Nice catch, the error showed after updating `@raycase/utils` to the latest version seems this is the reason, I'll follow your suggestions in the following commit

Reply:
Hi @sxn Thanks for your suggestion, They are pushed in the last commit, I tried to use `onError` but it does not allow using `ShowToast` with it due to expected types, so I used normal error checking here is a gif show the new behaviour  
https://share.cleanshot.com/2rS70sNNokkaDYEfC0Vo (couldn't upload it here it is above 10 Mb 😅 )

Reply:
you get that error because `(error) => showToast(Toast.Style.Failure, "Error")` is functionally equivalent to `(error) => { return showToast(Toast.Style.Failure, "Error") }`. You should not get any errors if you change it to `(error) => { showToast(Toast.Style.Failure, "Error") }`. 😄

Reply:
My bad 🤣 🤦‍♂️, thanks for clarifying things